{{ cookiecutter.package_name|slugify }}/docker-compose.yml

```diff
@@ -5,10 +5,6 @@ services:
     build:
       context: .
       target: dev
-      {%- if cookiecutter.private_package_repository_name %}
-      secrets:
-        - poetry_auth
-      {%- endif %}
```

Comment:
I think we need to keep these: these are build-time secrets that are used to access private packages when building the Docker image (a feature that was [recently introduced with Docker Compose v2.5.0](https://github.com/radix-ai/poetry-cookiecutter/pull/36)).